src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -68,17 +70,20 @@
  * A tool that generates {@code .d.ts} declarations from a Google Closure JavaScript program.
  */
 public class DeclarationGenerator {
-
-  private static final String INTERNAL_NAMESPACE = "ಠ_ಠ.clutz_internal";
-
-  // Comments in .d.ts and .js golden files starting with '//!!' are stripped.
-  public static final Pattern GOLDEN_FILE_COMMENTS_REGEXP = Pattern.compile("(?m)^\\s*//!!.*\\n");
-
+  static final DiagnosticType CLUTZ_OVERRIDDEN_STATIC_FIELD =
+      DiagnosticType.error("CLUTZ_OVERRIDDEN_STATIC_FIELD",
+          "Found a statically declared field that does not match the type of a parent field "
+              + "with the same name, which is illegal in TypeScript.\nAt {0}.{1}");
+  static final DiagnosticType CLUTZ_MISSING_TYPES = DiagnosticType.error("CLUTZ_MISSING_TYPES",
+      "The code does not compile because it is missing some types. This is often caused by the"
+          + "code that's referenced either missing dependencies, "
```

Comment:
odd sentence - `This is often caused by missing dependencies in the referenced code or missing externs in your build rule`